project_setup.py

```diff
@@ -45,7 +45,9 @@ def setup(
         )
         assert image_builder.deploy()
         default_image = image_builder.spec.image
-    project.set_default_image(project.get_param("default_image"))
+        project.set_default_image(default_image)
+    else:
+        project.set_default_image(project.get_param("default_image"))
```

Comment:
The code in the `if` statement is executed only if there is no default image as a parameter, so the `else` statement is redundant.